manly.py

```diff
@@ -154,4 +134,17 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+    parser = argparse.ArgumentParser(prog='manly',
+        description="Explain how FLAGS modify a PROGRAM's behaviour.",
```

Comment:
Should this say PROGRAM's instead of `command`? or the other way around?

Reply:
I agree with that. I believe that'd bash's convention - to call them commands rather than programs.